test/e2e/common/container_probe.go

```diff
@@ -107,9 +108,9 @@ var _ = framework.KubeDescribe("Probing container", func() {
 	})
 
 	/*
-		    Testname: pods-cat-liveness-probe-restarted
-		    Description: Make sure the pod is restarted with a cat /tmp/health
-			liveness probe.
+		Release : v1.9
+		Testname: Pod liveness probe, cat, restart
+		Description: A Pod is created with liveness probe that that uses ‘exec’ command to cat /temp/health file. The Container deletes the file /temp/health after 10 second, triggering liveness probe to fail. The Pod SHOULD now be killed and restarted incrementing restart count to 1.
```

Comment:
Nits:
- that that
- uses 'exec' command to cat /temp health file... this isn't the right wording, I'm not sure what is 100% correct, ie: if I go full API resources I could say "liveness probe that uses the ExecAction Handler to run the 'cat /temp/health' command"

- again, unclear on why SHOULD instead of MUST here

Reply:
fixed.